README.md

```
@@ -50,11 +50,11 @@ The following requirements were used for this project. Feel free to use other de
 
 
 1. Windows 10 OS
-2. NodeJS LTS v20 or higher
+2. NodeJS LTS v24 or higher
    ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)